extension/src/project/projectTree.ts

```diff
@@ -124,18 +124,25 @@ export class ProjectTreeProvider implements vscode.TreeDataProvider<DisplayNode>
 
     public updateData(newRootNode: ProjectNode) {
         this.rootNode = newRootNode;
-
+        vscode.commands.executeCommand("setContext", "autolisp.hasProject", ProjectTreeProvider.hasProjectOpened());
         this.onChanged.fire();
     }
 
     public refreshData(data?: DisplayNode) {
+        vscode.commands.executeCommand("setContext", "autolisp.hasProject", ProjectTreeProvider.hasProjectOpened());
```

Comment:
This method is also used to refresh a single LSP file node. I think it's enough to just do it in the "updateData" method, in which we truely change the rootNode field.